### ISSUE TITLE
Documentation missing or incomplete for many API/doc sections.

### ISSUE THREAD

**Issue by remy-luisant** [jgaeddert/liquid-dsp#327]
I have noticed that the website does not have the qdetector documented. There are others, too. Multichannel comes to mind.

I will not give the whole list here. Suffice to say, this has been a grave problem in quickly moving around the library. (I no longer need the details on the two, I'm merely reporting the lack of documentation as a former impediment.)

The problem spans both the doc and the api tabs. The two are also drifting out of sync with each other, it seems.

**Comment by jgaeddert** [comment]
Fair point, and I've been (slowly) trying to synchronize them and complete missing pieces. I can prioritize the `qdetector` object and multichannel module.
